pallets/msa/src/lib.rs

```diff
@@ -511,26 +548,8 @@ pub mod pallet {
 		#[pallet::call_index(2)]
 		#[pallet::weight(T::WeightInfo::create_provider(provider_name.len() as u32))]
 		pub fn create_provider(origin: OriginFor<T>, provider_name: Vec<u8>) -> DispatchResult {
```

Comment:
Maybe I am not understading. It looks like I can still register to be a provider without governance?

Reply:
We can call this in local an rococo.

Reply:
ahh I see the feature flag is on the origin. Why not put the feature flag on the function itself?

Reply:
@rlaferla @wilwade ^^ pls to check my explanation in case I missed something